libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -844,6 +847,7 @@ class MapboxNavigation(
 
         private const val USER_AGENT: String = "MapboxNavigationNative"
         private const val THREADS_COUNT = 2
+        private const val MIN_DAYS_TO_STORE_TILES = 0
```

Comment:
How does this work? It seems that we're saying that we're not storing tiles at all (`0`). Is that on purpose?

Reply:
this is the minimum time in days between the version on the server and the latest local one

Reply:
We were talking about having here a longer interval to minimize bandwidth consumption instead of always downloading the latest version.

What is a good target here @mskurydin @etl @jyrigo @truburt?

Reply:
Based on the update pace we have today, we would recommend setting the default to 14 days (2 weeks).

Reply:
Let us stick with 8 weeks as default, and state it clearly in changelog and docs that nav tiles update frequency will directly affect network consumption.